src/ranged.cpp

```diff
@@ -948,43 +943,42 @@ static void do_aim( player *p, std::vector <Creature *> &t, int &target,
         p->recoil = std::max(MIN_RECOIL, p->recoil);
     }
 
-    const int aim_amount = p->aim_per_time( *relevant, p->recoil );
+    const double aim_amount = p->aim_per_move( *relevant, p->recoil );
     if( aim_amount > 0 ) {
         // Increase aim at the cost of moves
-        p->moves -= 10;
+        p->moves--;
```

Comment:
> Aiming is either in 100 move blocks via pause

I can't find that change. Each '.' press was 1 invocation of `do_aim` and I don't see any new invocation of `do_aim` here.